datasets/de/bka_wanted/crawler.py

```diff
@@ -0,0 +1,124 @@
+from zavod import Context
+from zavod import helpers as h
+from xml.etree import ElementTree
+from normality import collapse_spaces
+
+
+def get_element_text(
```

Comment:
is this different from el.text_content() ?

Reply:
Not very, 
It helps to get all the text content from xpaths that has multiple node match.
Example for an xpath `//p` , it extracts the text contents from all `<p>` and joins them together

Reply:
Not very, 
It helps to get all the text content from xpaths that has multiple node match.
Example for an xpath `//p` , it extracts the text contents from all `<p>` and joins them together

Reply:
Not very, 
It helps to get all the text content from xpaths that has multiple node match.
Example for an xpath `//p` , it extracts the text contents from all `<p>` and joins them together